src/main/java/org/apache/datasketches/sampling/VarOptItemsSketch.java

```diff
@@ -785,6 +785,10 @@ T getItem(final int idx) {
     return weights_.get(idx);
   }
 
+  boolean isMarksValid() {
```

Comment:
why expose this?

Reply:
If you look at what we had before in VarOptItemsSketchTest Lines 522-528 :
```
    try {
      sketch.getMark(0);
      fail();
    } catch (final NullPointerException e) {
      // expected
    }
```
SpotBugs identified that we were catching a NullPointerException and then swallowing it.  That is a no-no.  Best practice is to check for null, but never actually "catch" a NullPointerException with a swallow.  We had a few of those.  Even though this was in test. It is still bad practice, so I fixed it.

I added isMarksValid() and changed the test to 
`assertFalse(sketch.isMarksValid());`

This avoids having to catch a NullPointerException and then swallow it.